### ISSUE TITLE
No Sound Output

### ISSUE THREAD

**Issue by DremOSDeveloperTeam** [ddf/Minim#122]
Minim is not outputting any sound. This may be related to whatever issue the author of #12 was having, but I doubt it since it's so old.

I don't see any audio outputs anywhere in any PulseAudio volume controls, such as pavucontrol.

Arch Linux
Processing 4.0.1 (from AUR)

**Comment by ddf** [comment]
Are you able to output sound using the built-in Processing sound classes?

**Comment by DremOSDeveloperTeam** [comment]
No, it doesn't. Sorry, I didn't think to try that before. Looks like it's probably an issue with Processing then.

**Comment by ddf** [comment]
Or maybe something to do with the JavaSound included with the Processing distro? I'm not sure if they still package a version of Java with the Linux distro of Processing, I haven't used Processing or Linux in a long while!

**Comment by DremOSDeveloperTeam** [comment]
Ah, I didn't even notice the last commit date!
